src/reporters/json-reporter.js

```diff
@@ -173,4 +173,8 @@ export default class JSONReporter extends BaseReporter {
   auditSummary(auditMetadata: AuditMetadata) {
     this._dump('auditSummary', auditMetadata);
   }
+
+  auditMute(mutedAdvisories: AuditAdvisory[]) {
+    this._dump(mutedAdvisories);
```

Comment:
```suggestion
    this._dump('auditMute', mutedAdvisories);
```

`type: string` is required for `_.dump`